### DISCUSSION TITLE
Teensy bootlooping

### DISCUSSION THREAD

**Discussion by Fidexy** [ZeroZeroOne-dev/00Key#16] [General]
I've got the code compiled and uploaded to the board, but the entire thing keeps bootlooping every few seconds or so

[video](https://youtu.be/KlvJpC_XKpw)

**Comment by ZeroZeroOne-dev** [comment]
if you dont have underglow you might need to comment out the `Underglow::setup();` and `Underglow::loop();` lines in main.cpp
